src/unix/freebsd.c

```diff
@@ -287,3 +287,18 @@ int uv__recvmmsg(int fd, struct uv__mmsghdr* mmsg, unsigned int vlen) {
   return errno = ENOSYS, -1;
 #endif
 }
+
+ssize_t
+uv__fs_copy_file_range(int fd_in,
+                       off_t* off_in,
+                       int fd_out,
+                       off_t* off_out,
+                       size_t len,
+                       unsigned int flags)
+{
+#if __FreeBSD_version >= 1300000
```

Comment:
Shouldn't this be something like `#if __FreeBSD__ >= 13 && !defined(__DragonFly__)`, like the other guards in this files? If not, why not?

Reply:
true but it s shorter at least but no pb I can change.